contracts/token/ERC20/StandardBurnableToken.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.4.18;
+
+import "./BurnableToken.sol";
+import "./StandardToken.sol";
+
+/**
+ * @title Standard Burnable Token
+ * @dev Adds burnFrom method to ERC20 implementations
+ */
+contract StandardBurnableToken is BurnableToken, StandardToken {
+
+  /**
+   * @dev Burns a specific amount of tokens from the target address and decrements allowance
+   * @param _from address The address which you want to send tokens from
+   * @param _value uint256 The amount of token to be burned
+   */
+  function burnFrom(address _from, uint256 _value) public {
+    require(_value <= allowed[_from][msg.sender]);
+    allowed[_from][msg.sender] = allowed[_from][msg.sender].sub(_value);
```

Comment:
See #707. No need to do anything about it now but we should keep it in mind if we move forward with it.